qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1293,13 +1296,13 @@ def count_ops(self):
                 op_dict[name] += 1
         return op_dict
 
-    def count_ops_longest_path(self):
-        """Count the occurrences of operation names on the longest path.
+    def count_ops_deepest_path(self):
+        """Count the occurrences of operation names on the deepest path.
```

Comment:
Same here [as before](https://github.com/Qiskit/qiskit-terra/pull/3488/files#r362342902).

Reply:
same as in above comment